app/scripts/modules/core/src/pipeline/status/Artifact.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+
+import { IArtifact } from 'core/domain';
+import { ArtifactIconService } from 'core/artifact';
+
+import './artifact.less';
+
+export interface IArtifactProps {
+  artifact: IArtifact;
+  isDefault?: boolean;
+  sequence?: number;
+}
+
+export interface IArtifactState {}
```

Comment:
Can probably just omit this empty interface, and the constructor

Reply:
Great point, I didn't realize you could omit state entirely, fixed